docs/minio-client-complete-guide.md

```diff
@@ -862,6 +863,128 @@ myobject.txt:    14 B / 14 B  ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓
 Waiting move operations to complete
 ```
 
+<a name="snap"></a>
+### Command `snap`
+
+`snap` command generates snapshots of S3 data at a given date/time. It relies on S3 bucket versioning feature to pick objects versions.
```

Comment:
```suggestion
`snap` command generates snapshots of S3 metadata at a given date/time. It relies on S3 bucket versioning feature to pick objects versions.
```

We do not in any way want to indicate that it stores actual data (for now at least).

Reply:
Could you add a sentence on what it can be used for?